src/menu/menu.c

```diff
@@ -1030,6 +1030,13 @@ init_rootmenu(struct server *server)
 	if (!menu) {
 		current_menu = NULL;
 		menu = menu_create(server, "root-menu", "");
+
+		current_item = item_create(menu, _("Terminal"), false);
+		fill_item("name.action", "Execute");
+		fill_item("command.action", "lab-sensible-terminal");
```

Comment:
I think we need to do [some things](https://github.com/labwc/labwc/blob/master/CONTRIBUTING.md#coders) here when we add a new translation item.